docs/source/install/__packages_beta_warning.rst

```diff
@@ -0,0 +1,7 @@
+.. warning::
+
+   Packages based installation is currently is beta and upgrades are not yet
```

Comment:
I'm open to wording this differently or removing this notice all together.

The fact is that AIO is out of date and not supported so we need to point all the new users to the package based installation instructions and this warning might scare some people off...